ted_sws/core/adapters/cmd_runner.py

```diff
@@ -112,11 +122,36 @@ def on_end(self):
             now=str(self.end_time),
             time=self.end_time - self.begin_time
         ))
+        self.exit(self.exit_code)
+
+    @classmethod
+    def exit(cls, exit_code=DEFAULT_EXIT_CODE):
+        if exit_code > EXIT_CODE_OK:
+            sys.exit(exit_code)
 
 
 class CmdRunnerForMappingSuite(CmdRunner):
     repository_path: Path
+    mapping_suite_id: str
+    notice_id: List[str]
 
     def is_mapping_suite(self, suite_id):
         suite_path = self.repository_path / Path(suite_id)
         return os.path.isdir(suite_path) and any(f == MS_METADATA_FILE_NAME for f in os.listdir(suite_path))
+
+    @classmethod
+    def _init_list_input_opts(cls, input_val) -> List:
+        input_set = OrderedSet()
+        if input_val and len(input_val) > 0:
+            for item in input_val:
+                input_set |= OrderedSet(map(lambda x: x.strip(), item.split(",")))
+        return list(input_set)
+
+    def skip_notice(self, notice_id: str) -> bool:
+        return self.notice_id and len(self.notice_id) > 0 and notice_id not in self.notice_id
+
+    def run_cmd(self):
+        if self.mapping_suite_id:
+            self.log(LOG_WARN_TEXT.format("MappingSuite: ") + self.mapping_suite_id)
+        if self.notice_id and len(self.notice_id) > 0:
```

Comment:
In python in if construction values as : "", [], (). {}, None, 0, all of them return false. Use just if self.notice_id instead of current if

Reply:
it's more clear this way (not python related), to not become programming language dependent :D but for your peace, I'll change it